### ISSUE TITLE
Typescript scaffold

### ISSUE THREAD

**Issue by AdsonCicilioti** [dgwyer/create-wp-block#10]
Hi man! Nice tool!

Is there a possibility of a scaffold for Typescript?

Tailwind support (described in the README) will be super welcome too.

**Comment by dgwyer** [comment]
Yes, it's a possibility but not a priority for me right now so I can't say if/when this would be added. But it's something I'm interested in too.
